src/modules/esl-tooltip/core/esl-tooltip.ts

```diff
@@ -43,21 +45,17 @@ export class ESLTooltip extends ESLPopup {
     return document.createElement('esl-tooltip');
   }
 
-  /** List of all focusable elements inside Tooltip */
-  public get focusableElements(): Element[] {
-    return getKeyboardFocusableElements(this);
-  }
-
-  /** First focusable element inside Tooltip */
-  public get firstFocusableElement(): Element | null {
-    const els = this.focusableElements;
-    return els.length ? els[0] : null;
+  /** List of all focusable elements inside instance */
+  public get $focusables(): HTMLElement[] {
+    return getKeyboardFocusableElements(this) as HTMLElement[];
   }
 
-  /** Last focusable element inside Tooltip */
-  public get lastFocusableElement(): Element | null {
-    const els = this.focusableElements;
-    return els.length ? els[els.length - 1] : null;
+  /** First and last focusable elements inside instance */
+  public get $boundaryFocusable(): {$first: HTMLElement | undefined, $last: HTMLElement | undefined} {
```

Comment:
mb we can discuss to move it to helpers as the same focus loop is on the modal

Reply:
sorry for misunderstanding
previously it was discussed to leave this functionallity localy until focus trap will be implemented